homework.2 (Extra)/homework.2.1/main.swift

```diff
@@ -0,0 +1,66 @@
+// homework.2 (Extra exercises)
+
+import Foundation
+
+// Task 1
+
+let height = 35
+let width = 25
+
+height > width ? print("The number \(height) is greater than \(width)") : print("The number \(width) is greater than \(height)")
+
+// Task 2
+
+let hasKey = false
+let hasDocuments = true
+let hasPen = true
+let takenApple = false
+let takenOrange = true
+var shouldGoToWork = (hasKey && hasDocuments && hasPen) && (takenApple || takenOrange)
+
+shouldGoToWork ? print("We didn't forget anything :)") : print("Look, maybe you forgot something?")
+
+// Task 3
+
+var valueOne: Double = 1
+
+if valueOne.truncatingRemainder(dividingBy: 3) == 0 && valueOne.truncatingRemainder(dividingBy: 5) == 0 {
+    print("FizBuz")
+}
+else if valueOne.truncatingRemainder(dividingBy: 5) == 0 {
+    print("Fiz")
+}
+else if valueOne.truncatingRemainder(dividingBy: 3) == 0 {
+    print("Buz")
+}
+else {
+    print("The number does not satisfy any of the conditions, try again :)")
+}
+
+// Task 4
+
+var yearsOldRange = 16..<18
+
+yearsOldRange.upperBound > 18 ? print("You are allowed to drink") : print("Drink juice")
+
+if !(yearsOldRange.contains(18)) {
+    print("Think about a way how you can look like an adult")
+}
```

Comment:
You don't pass the age as a variable to check here. So this code will always return the same result